packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -97,6 +97,7 @@ export function generateRuntimeOptions(
 		enableOpReentryCheck: [true],
 		// Compressed payloads exceeding this size will be chunked into messages of exactly this size
 		chunkSizeInBytes: [204800],
+		enableGroupedBatching: [undefined],
```

Comment:
Can we enable testing of that feature from the start?

Reply:
We should also do `[true, false]` here to get a mix of clients who do and who don't, in order catch any possible regressions early.